### ISSUE TITLE
Get NPM assets included via composer

### ISSUE THREAD

**Issue by jrhodes-sitback** [Sitback/presto#67]
See:

- https://packagist.org/packages/fxp/composer-asset-plugin
- https://asset-packagist.org/

Use 

Add repository:

```json
        {
            "type": "composer",
            "url": "https://asset-packagist.org"
        }
```

Require slick-carousel

```json
    "require": {
        "bower-asset/slick-carousel": "^1.6",
```

**Comment by chinthakagodawita** [comment]
@jrhodes-sitback Fixed in the `feature/67--add-npm-assets` branch. Could you merge into your carousel branch and tesct?

Main changes:
* Added `oomphinc/composer-installers-extender`
* Turned Composer package sorting off (the above package needs to be installed **before** any of the asset packages)
* Added `"type:bower-asset"` and `"type:npm-asset"` to `installer-paths`

**Comment by chinthakagodawita** [comment]
Will be fixed as part of #50
